### ISSUE TITLE
Database issues today

### ISSUE THREAD

**Issue by jussi-club-soda** [club-soda/club-soda-guide#281]
Something odd going on in production:

When adding new drinks, they don't seem to have the brand attached (search for Limca for example to see what I mean)

Anja said she can't add venues at all

@iteles @Cleop ?

**Comment by iteles** [comment]
@jussi-club-soda The drinks side of things looks like it might have some link to #279, @SimonLab and @Cleop are on it 👍

**Comment by Cleop** [comment]
All drinks added today have a brand of `NULL`.

Limca/Thums up/ Aranciata should be linked to the Coca Cola Company (241) but it is not so its brand is `NULL`. Other than that all other fields about Limca look as they should.

When I artificially add Coca Cola and give Limca a weighting, I can view Limca on the landing page.

**Comment by Cleop** [comment]
Error when you try to edit the brand of a drink:

![image](https://user-images.githubusercontent.com/16775804/50347668-41146180-052e-11e9-87fe-a946d94e5fbb.png)

**Comment by Cleop** [comment]
**Issues #281 and #279 are the same problem which is:

- you can't edit anything about an existing drink without an error and you can't create new drinks without an error.

The error is the :drink_types error in the screenshot above.**

**Comment by Cleop** [comment]
When you try to create a venue you get caught by the error as venue_type can't be blank. I inspected the venue_params and they don't include venue_type:
```
%{
  "address" => "",
  "city" => "",
  "description" => "",
  "facebook" => "",
  "favourite" => "false",
  "instagram" => "",
  "phone_number" => "",
  "postcode" => "PO20 8LO",
  "twitter" => "",
  "venue_name" => "test",
  "website" => ""
}
```

**Comment by Cleop** [comment]
It seems as though the version of alog is not right or doesn't contain the latest code. This is what the app has access to:
![image](https://user-images.githubusercontent.com/16775804/50351893-6f4c6e00-053b-11e9-95e4-3f062bc2a008.png)
This is what is in the alog repo:
```
    def insert(struct_or_changeset) do
        case check_for_unique_index() do
          :ok ->
            struct_or_changeset
            |> insert_entry_id()
            |> apply_constraints()
            |> @repo.insert()

          {:error, msg} ->
            raise msg
        end
      end
```

**Comment by Cleop** [comment]
`name='venue[venue_types][Pubs]'` << The first 'venue' must be lowercase, it is currently uppercase.

**Comment by jussi-club-soda** [comment]
Whatever is going on with this, also seems to stop the drinks showing on the site. I've deleted every drink that was added yesterday (all had no brand as far as I can tell) and we're back to normal in terms of the home page carousel and the drinks page.

**Comment by SimonLab** [comment]
@jussi-club-soda I've just open a new PR (https://github.com/club-soda/club-soda-guide/pull/284) which is going to resolve this issue. Once this one is merged I was going to update the new created drinks but I guess they can be recreated (this can also be used to test again that everything is working well). I'll then update the weighting issue.

**Comment by jussi-club-soda** [comment]
I think this is working again!
